tests/kubernetes-agent/KubernetesAgent.IntegrationTests/Setup/Common/AssemblyExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Reflection;
+
+namespace Octopus.Tentacle.Kubernetes.Tests.Integration.Support;
```

Comment:
namespace should be `KubernetesAgent.Integration....`

Reply:
Done!